Documentation

Hi, would you add some documentation to help those who are new to NDI to get this up and running on their PI4?

Besides `git`, what other dependencies do I need to install to get `build.sh` working?

```
$ bash build.sh 
src/main.cpp:17:10: fatal error: libconfig.h++: No such file or directory
 #include <libconfig.h++>
          ^~~~~~~~~~~~~~~
compilation terminated.
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
 .DS*
 build
 .vscode/
+.idea/

--- a/README.md
+++ b/README.md
@@ -1,5 +1,4 @@
-RaspiNDI
-========
+# RaspiNDI
 
 RaspiNDI is a simple NDI send library for Raspberry Pi. It was initially
 developed to allow a streaming setup for a church during the COVID-19 outbreak
@@ -7,3 +6,70 @@ in 2020.
 
 It uses the NDI library, allowing for auto-discovery of streams on the local
 network.
+
+## Getting started
+
+These intructions are for a clean installation of
+[Raspberry Pi OS](https://www.raspberrypi.org/software/).
+All steps are performed on the command line.
+
+### tl;dr
+
+- Clone this repo.
+- `./easy-setup.sh`
+- `/opt/raspindi/raspindi.sh`
+
+### Step by step
+
+Make sure git is installed.
+
+```
+sudo apt update
+sudo apt install git
+```
+
+Clone this repository and `cd` into it.
+
+```
+git clone https://github.com/raspberry-pi-camera/raspindi.git
+cd raspindi
+```
+
+Install compilation dependencies.
+
+```
+sudo apt update
+sudo apt install libconfig++-dev
+```
+
+Compile.
+
+```
+./build.sh
+```
+
+Install.
+
+```
+sudo ./install.sh
+```
+
+Install runtime dependencies.
+
+```
+sudo apt update
+sudo apt install libjasper-runtime
+```
+
+Run it. (It does not require root to run.)
+
+```
+/opt/raspindi/raspindi.sh
+```
+
+Open an NDI receiver somewhere on the same network. It should detect the 
+Raspberry Pi camera after a few seconds.
+
+[OBS Studio](https://obsproject.com/) with the 
+[OBS-NDI plugin](https://github.com/Palakis/obs-ndi/releases/) 
+works well.

--- a/build.sh
+++ b/build.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env sh
 
 if [ ! -d "build" ]; then
   mkdir build

--- a/easy-setup.sh
+++ b/easy-setup.sh
@@ -1,0 +1,10 @@
+#!/usr/bin/env sh
+
+set -eu
+
+sudo apt update
+sudo apt install libconfig++-dev libjasper-runtime
+./build.sh
+sudo ./install.sh
+
+echo "Done"

--- a/install.sh
+++ b/install.sh
@@ -1,0 +1,21 @@
+#!/usr/bin/env sh
+
+INSTALL_DIR="/opt/raspindi"
+BIN_DIR="$INSTALL_DIR/bin"
+LIB_DIR="$INSTALL_DIR/lib"
+
+mkdir "$INSTALL_DIR"
+mkdir "$LIB_DIR"
+mkdir "$BIN_DIR"
+
+cp lib/ndi/* "$LIB_DIR"
+cp lib/raspicam/* "$LIB_DIR"
+
+cp build/raspindi "$BIN_DIR"
+
+cp etc/raspindi.conf.default /etc/raspindi.conf
+
+echo '#!/usr/bin/env sh
+LD_LIBRARY_PATH="/opt/raspindi/lib" /opt/raspindi/bin/raspindi
+' > "$INSTALL_DIR/raspindi.sh"
+chmod +x "$INSTALL_DIR/raspindi.sh"

--- a/uninstall.sh
+++ b/uninstall.sh
@@ -1,0 +1,6 @@
+#!/usr/bin/env sh
+
+INSTALL_DIR="/opt/raspindi"
+
+rm -rf "$INSTALL_DIR"
+rm -f /etc/raspindi.conf

